core/domain/story_services_test.py

```diff
@@ -557,31 +568,33 @@ def test_update_story_schema(self):
         # Check version is updated.
         self.assertEqual(new_story_dict['version'], 2)
 
-        # Delete version and check that the two dicts are the same.
-        del orig_story_dict['version']
-        del new_story_dict['version']
+        # Instead of Deleting the version key, we are making them equal to
+        # check if the other contents of the two dicts are same or not.
```

Comment:
```suggestion
        # Instead of deleting the version key, we are making them equal to
        # check if the other contents of the two dicts are the same or not.
```

Reply:
Done!